mllib/src/main/scala/org/apache/spark/ml/feature/ChiSqSelector.scala

```diff
@@ -44,6 +44,7 @@ import org.apache.spark.sql.types.StructType
  * By default, the selection method is `numTopFeatures`, with the default number of top features
  * set to 50.
  */
+@deprecated("use UnivariateFeatureSelector instead", "3.1.0")
```

Comment:
Should we change "3.1.0" to "3.1.1"?

Reply:
ping @HyukjinKwon , we are trying to put this pr in 3.1. Should we use '3.1.0' here?

Reply:
Oh it should be 3.1.1